timers.js

```diff
@@ -1,4 +1,45 @@
+//1
+var start = document.getElementById("start");
+var pause = document.getElementById("pause");
+var reset = document.getElementById("reset");
 
+//3
+var seconds= 0;
+var timerId;
 
 
+  //2
+// start.addEventListener("click", function() {
+//   });
+//       pause.addEventListener("click", function() {
+//         });
+//             reset.addEventListener("click", function() {
+//
+//             });
 
+  //4
+          var updateTime =  function() {
+            seconds++;
+            // document.write(seconds)
```

Comment:
Looks like you have the right idea, display the results each time you update the timer.